science/netcdf/Portfile

```diff
@@ -10,7 +10,7 @@ PortGroup                   legacysupport 1.1
 # strnlen
 legacysupport.newest_darwin_requires_legacy 10
 
-github.setup                Unidata netcdf-c 4.8.1 v
+github.setup                Unidata netcdf-c 4.9.0 v
 revision                    1
```

Comment:
sorry for the delay @remkos ! The `revision` field needs to be reset to `0` when updating the version; if you can make that change I will merge this right away.